ci/Jenkinsfiles/build.groovy

```diff
@@ -67,7 +67,7 @@ String getCurrentVersion() {
 }
 
 String getReleaseVersion() {
-  container('maven') {
+  container('ftests') {
```

Comment:
`DEFAULT_CONTAINER`, which can be defined as a global constant at the very top of the Grrovy file, as I suspect it doesn't work here currently since it is an env var and this function is called from the `environment` block itself

Reply:
Ah yes, that's why I didn't use the `DEFAULT_CONTAINER` here, but I'll extract it to a global variable and use. Thanks.